compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/diagnostics/DiagnosticErrorCode.java

```diff
@@ -267,10 +267,10 @@
     ERROR_INVALID_METADATA("BCE0621", "error.invalid.metadata"),
     ERROR_INVALID_QUALIFIER("BCE0622", "error.invalid.qualifier"),
     ERROR_ANNOTATIONS_ATTACHED_TO_STATEMENT("BCE0623", "error.annotations.attached.to.statement"),
-    ERROR_MORE_FIELD_MATCH_PATTERNS_AFTER_REST_FIELD("BCE0624", "error.more.field.match.patterns.after.rest.field"),
+    ERROR_FIELD_MATCH_PATTERN_AFTER_REST_FIELD("BCE0624", "error.field.match.pattern.after.rest.field"),
     ERROR_ACTION_AS_A_WAIT_EXPR("BCE0625", "error.action.as.a.wait.expr"),
     ERROR_INVALID_USAGE_OF_VAR("BCE0626", "error.invalid.usage.of.var"),
-    ERROR_MORE_MATCH_PATTERNS_AFTER_REST_MATCH_PATTERN("BCE0627", "error.more.match.patterns.after.rest.match.pattern"),
+    ERROR_MATCH_PATTERN_AFTER_REST_MATCH_PATTERN("BCE0627", "error.match.pattern.after.rest.match.pattern"),
```

Comment:
Do we need both this and `ERROR_FIELD_MATCH_PATTERN_AFTER_REST_FIELD`?

Reply:
+1 let's keep only one